x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/KibanaOwnedReservedRoleDescriptors.java

```diff
@@ -456,11 +456,13 @@ static RoleDescriptor kibanaSystem(String name) {
                         TransportUpdateSettingsAction.TYPE.name()
                     )
                     .build(),
+
                 RoleDescriptor.IndicesPrivileges.builder().indices("risk-score.risk-*").privileges("all").build(),
                 RoleDescriptor.IndicesPrivileges.builder()
                     .indices(".asset-criticality.asset-criticality-*")
-                    .privileges("create_index", "manage", "read")
+                    .privileges("create_index", "manage", "read", "write")
```

Comment:
Heya @machadoum ! We don't usually want `kibana_system` to have `write` access to system indexes not owned by Kibana.

Could you please provide more details about why this increase in permissions is necessary?

Reply:
Ah ok so it's for a migration; do you think this permission could eventually be removed in 9.x after all serverless projects have upgraded? It could continue to exist in 8.latest so clusters upgrading would still get the migration done.

Reply:
hmm, let me think.

In that case, what would happen if an ESS cluster upgrades directly from 8.15 to 9.x?

Reply:
I think the process has been that clusters have to go to prev_major.latest before going to the new major, but that was 7.17.latest before 8.x, how can we check if that will still apply?

We could always create an issue to revisit removing it later, what do you think?

Reply:
That makes sense to me. We have plenty of time to remove it from 9.x.

Reply:
Yes that sounds good to me.